adaptive_memory_v3.1.py

```
@@ -4303,11 +4560,129 @@ async def _get_embedding(self, text: str) -> Optional[np.array]:
+                    f"Re-embedding produced unexpected dimension {new_embedding.shape[0]}, expected {expected_dim}"
+                )
+                return None
+            else:
+                logger.warning(f"Failed to re-embed memory {memory_id}")
+                return None
+
+        except Exception as e:
+            logger.error(f"Error re-embedding memory {memory_id}: {e}\n{traceback.format_exc()}")
+            return None
+
+    async def _store_embedding_in_db(self, user_id: str, memory_id: str, memory_text: str, embedding: np.array) -> None:
```

Ruff (0.14.10)

Unused method argument: `memory_text`

(ARG002)